src/main.rs

```diff
@@ -94,7 +93,7 @@ async fn main() -> Result<(), ()> {
     // TODO: what if IF_ID_IN and IF_ID_OUT are different?
     let conn_if_id: u32 = misc::find_env("PLUTO_IF_ID_IN")?
         .parse::<u32>()
-        .map_err(|e| error! {"parse if_id failed: {}", e})?;
```

Comment:
Yes, we should wrap main to handle the error too. Will update later.

Originally I introduced eyre so that I could look into what the error was and took action on it, but I forgot....

Reply:
lgtm